documentation/source/simulator_support.rst

```diff
@@ -256,6 +256,9 @@ If you try to use them with :term:`FLI`, you will see a ``vsim-FLI-3155`` error:
 
 ModelSim DE and SE (and Questa, of course) support the :term:`FLI`.
 
+In order to start ModelSim or Questa with the graphical interface and for the simulator to remain active after the tests have completed, set :make:var:`GUI=1`.
+If you have previously launched a test without this setting you might have to delete the ``sim_build`` directory to get the correct behavior.
```

Comment:
```suggestion
If you have previously launched a test without this setting, you might have to delete the ``sim_build`` directory to get the correct behavior.
```
maybe? Native speaker required ;)

Reply:
Sounds good to me - I think it is referred to as an Introductory Phrase which should be a good use of commas.